.docker/build-helper.Dockerfile

```diff
@@ -1,7 +1,6 @@
 # we use debian:testing due to paho otherwise we would need to install
```

Comment:
```suggestion
# we use debian:stable due to paho otherwise we would need to install
```

Reply:
removed comment; because stable is actually a our target therefore it doesn't need a comment explaining why :)